ChessCrafterDesktop/src/main/resources/GuiStrings_en_US.properties

```diff
@@ -130,6 +130,9 @@ Driver.startNewGame=Start a new Game
 Driver.sureYouWannaQuit=Are you sure you want to Quit?
 Driver.variants=Variants
 Driver.viewCompleted=View Completed Game
+FileUtility.savedGameExtension=.chesscrafter
+FileUtility.craftedGameExtension=.craftconfig
+FileUtility.pieceExtension=.piece

```

Comment:
Our file extensions aren't going to change depending on your language, so they shouldn't be localizable strings. Just make them private static final constants in FileUtility